packages/earth-map/src/index.tsx

```diff
@@ -30,6 +32,30 @@ import { GTM_TAG } from './config';
 import auth0 from './config/auth0';
 import * as serviceWorker from './serviceWorker';
 
+import translationEN from 'locales/en/translation.json';
+import translationES from 'locales/es/translation.json';
+import translationFR from 'locales/fr/translation.json';
+
+i18n.use(initReactI18next).init({
+  resources: {
+    en: {
+      translation: translationEN,
+    },
+    es: {
+      translation: translationES,
+    },
+    fr: {
+      translation: translationFR,
+    },
+  },
+  lng: 'en',
+  fallbackLng: 'en',
+  interpolation: {
+    escapeValue: false,
+  },
+  keySeparator: false,
+});
+
```

Comment:
Please move this code in `src/locales/index.ts` and just import it here.
Also, make an enum ELang with language keys to be used throughout the app.
```suggestion
import translationEN from 'locales/en/translation.json';
import translationES from 'locales/es/translation.json';
import translationFR from 'locales/fr/translation.json';

i18n.use(initReactI18next).init({
  resources: {
    [Elang.EN]: {
      translation: translationEN,
    },
    [Elang.ES]: {
      translation: translationES,
    },
    [Elang.FR]: {
      translation: translationFR,
    },
  },
  lng: Elang.EN,
  fallbackLng: Elang.EN,
  interpolation: {
    escapeValue: false,
  },
  keySeparator: false,
});

```